README.md

```diff
@@ -46,7 +46,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:4.10.0-SNAPSHOT'
+    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:5.1.0-SNAPSHOT'
```

Comment:
Normally after releasing a major version, the `SNAPSHOT` is bumped to the next major version i.e. `6.0.0-SNAPSHOT`